plop-templates/Recipe/Recipe.tsx.hbs

```diff
@@ -16,8 +16,7 @@ export const {{pascalCase name}} = ({
   className,
   ...other
 }: Props) => {
-  const componentClassName = clsx(styles['{{dashCase name}}'], className, {
-  });
+  const componentClassName = clsx(styles['{{dashCase name}}'], className);
```

Comment:
Reordering import according to lint and removing empty block in `clsx()`, not related to DataSummaryCard